src/libraries/Microsoft.PowerFx.Json/Functions/JsonFunctionImpl.cs

```diff
@@ -319,7 +319,76 @@ public void Visit(TimeValue timeValue)
 
                 public void Visit(UntypedObjectValue untypedObjectValue)
                 {
-                    throw new ArgumentException($"Unable to serialize type {untypedObjectValue.GetType().FullName} to Json format.");
+                    Visit(untypedObjectValue.Impl);
+                }
+
+                private void Visit(IUntypedObject untypedObject, int index = 0)
+                {
+                    FormulaType type = untypedObject.Type;
+
+                    if (index > 40)
```

Comment:
>40 [](http://example.com/codeflow?start=32&length=2)

can you hoist to a const field?  We shouldn't have "magic" numerical literals in the code like this.

I'm sure at some future point, we'll want this to come from some config object, but fixed is fine now.  

FWIW - 40 is probably very high.

Reply:
I've used 20 instead